src/views/WorkspaceDetail.vue

```diff
@@ -177,7 +177,9 @@ export default Vue.extend({
     DeleteTableDialog,
     DownloadDialog,
   },
-  props: ['workspace', 'title'],
+  props: {
+    workspace: String as PropType<string>,
+  },
```

Comment:
Did you lose `title` here? Or is it not needed any more?

Reply:
It's never used in the code so I removed it